test/unit-test/ota_utest.c

```diff
@@ -1213,6 +1215,35 @@ void test_OTA_ProcessJobDocumentInvalidJson()
     TEST_ASSERT_EQUAL( OtaAgentStateWaitingForJob, OTA_GetState() );
 }
 
+/* Test that the job is rejected if the file key signature cannot be
+ * decoded. */
```

Comment:
Want to make these doxygen style comments with @brief to be consistent with the other tests with descriptions? This would apply to the rest of the new tests as well.